cumulus_library/cli_parser.py

```diff
@@ -297,4 +297,10 @@ def get_parser() -> argparse.ArgumentParser:
     add_target_argument(markdown)
     add_verbose_argument(markdown)
 
+    # Get study version
+
+    study_version = actions.add_parser("study-version", help="Gets the version of a specific study")
```

Comment:
This is fine. But I would vote for a single command that shows all known versions it can find. Like... if `--version` (and/or a `version` or `debug` subcommand) gave output like:
```
$ cumulus-library --version
Cumulus Library: 3.0.0
Python: 3.11.1
Studies:
  covid_symptom: 2.1.0
  ...etc...
  umls: 1.2.0
```

Since we register known studies in the Library, we can just iterate over that (plus any extra targets). You could throw in paths to the study modules if you wanted there too.

In my mind, this has some advantages:
- Easier to tell a user how to do it: "run --version" vs "run study-version -t covid_symptom -t core"
- Less error prone for the user
  - harder to forget a `-t core` or similar argument that we'd want to see, everything is built-in
  - or a situation where they run a `study-version -t covid_symptom` line they saved from us a week ago instead of the new `study-version -t umls` command we just gave them.
- Easier to add future useful bits, like the version of some major dependency like Jinja or whatever.

But this approach works too - I'm just brainstorming.

Reply:
i buy this (as long as I alpha sort the keys for lookup)

Reply:
ok, this is now the behavior.